phpseclib/Net/SSH2.php

```diff
@@ -2533,6 +2539,42 @@ public function exec(string $command, callable $callback = null)
         }
     }
 
+    /**
+     * Opens a channel
+     */
+    protected function openChannel(string $channel, bool $skip_extended = false): bool
+    {
+        $this->channelCount++;
+
+        if ($this->channelCount > 1 && $this->errorOnMultipleChannels) {
+            throw new \RuntimeException("Ubuntu's OpenSSH from 5.8 to 6.9 doesn't work with multiple channels");
```

Comment:
A little awkward to presume that errorOnMultipleChannels=true due to OpenSSH version, in consideration of the flag's generic name and possible future expansion of its use. I'd error with the more generic e.g. "Multiple channel feature has been disabled" or something to that effect.

Reply:
The more generic error message you propose doesn't really explain the situation either. Like anyone affected by the issue is liable to be like "well can you enable the feature?" as tho _I'm_ the reason it's not working when, in reality, it's not working because of a buggy modification that the Ubuntu devs made to their fork of OpenSSH.

Sure, the variable name could be made less generic but, then again, there is something to be said about not having a variable name that's not 1KB in size as well lol. Like maybe I could have it named `$this->errorOnMultipleChannelsBecauseUbuntuForkOfOpenSSH58-69IsBroken` but that introduces it's own can of worms with it being impractically long.